src/ansys/mechanical/core/run.py

```diff
@@ -35,7 +35,7 @@
 from ansys.mechanical.core.embedding.appdata import UniqueUserProfile
 from ansys.mechanical.core.feature_flags import get_command_line_arguments, get_feature_flag_names
 
-DRY_RUN = False
+DRY_RUN = False  # control this with an env var to opt into dry run?
```

Comment:
delete the comment here ?
```suggestion
DRY_RUN = False

Reply:
Or at least mark it as a TODO... ``# TODO: ...``

Reply:
Ah, I forgot to delete this comment